code/modules/antagonists/abductor/machinery/camera.dm

```diff
@@ -115,6 +115,11 @@
 	var/mob/living/carbon/human/C = owner
 	var/mob/camera/ai_eye/remote/remote_eye = C.remote_control
 	var/obj/machinery/abductor/pad/P = target
+
+	if(istype(get_area(remote_eye), /area/ai_monitored/turret_protected/ai))
```

Comment:
Would be cleaner if this was a flag of the area itself, rather than an explicit check.

Reply:
It is pure copy-pasta of the existing check just to fix an oversight.

I welcome another PR with those improvements which closes this one that also fixes the problem if you are so inclined.

Reply:
Yeah I would have used a list except abductors teleporting is only a problem for this one specific area, they're SUPPOSED to be essentially all-reaching, but them being able to warp right next to the AI core caused gameplay problems. It was a simple, low-quality fix for a problem that had been around for years.

Reply:
> Yeah I would have used a list except abductors teleporting is only a problem for this one specific area, they're SUPPOSED to be essentially all-reaching, but them being able to warp right next to the AI core caused gameplay problems. It was a simple, low-quality fix for a problem that had been around for years.

I mean, as long as they're beaming in there to prank the AI instead of kill it (as ayys aren't allowed to kill stuff except in self-defence), I don't really see the problem, and I'm saying this as a silicon main.

Reply:
I can't speak for all abductor teams, but the ones I dealt with just yesterday who figured out how to **Send Self* with an intellicard into the AI chamber were 100% planning to kill me as their very first task on the station, and they only failed in the end because they were trying to show off (and probably the gods intervened).

Silicon main here too, so while I agree these snowflakey checks for odd cases are not great and could be handled with policy I also understand that it is a grey area when it comes to abductors and making it harder for them to yoink and basically round remove the AI mechanically means admins have one less thing to worry about. There are many other ways including brainwashing / mind controlling a human to kill it, uploading self termination laws, etc. that they can use to take care of a pesky AI that is getting in the way. Hell if they just want an AI to play with abduct a Robo or the RD and make them build you one for you to steal.

I would actually like to see more AI subversion tactics employed by the advanced aliens that can look like anyone they scan including the Captain and RD and teleport almost anywhere and hack with ease. They could also build an illicit upload terminal with a little more effort than it takes to steal an intellicard.

Overall I am indifferent on the change and this PR is only meant as a **fix** not a **rebalance** because the intent to block all abductors teleports to the AI chamber was already codified.

Reply:
fair enough

Reply:
In the original PR I linked to a policy thread, where the decision was made that it was a code issue, and that abductors were absolutely allowed to murder an AI if they want. I emphatically agree that it'd be preferable if the abductors had tools to brainwash silicons like they do with humans, however.